site/docs/v1/tech/apis/_two-factor-disable-request-body.adoc

```diff
@@ -0,0 +1,16 @@
+==== Request Body
+
+[.api]
+
+[field]#applicationId# [type]#[UUID]# [optional]#Optional#::
+The application id for email templates and events.
```

Comment:
This doesn't quite explain what this is useful for.

Can you explain a bit more? Like, why would I pass the `applicationId` parameter?

Also, we captialize "Id"

Reply:
@mooreds the `applicationId` is used for logging events and allow the usage of application-specific email templates for MFA, if configured at the application level. I tested this by including the `applicationId` at the top level of the request, and configured my custom application-specific email MFA template but the tenant application was used so this might be a bug.

Reply:
@mooreds - as discussed a section on how to override the Tenant email template, in favor of an application specific one will have to be documented, as well as one for forgot password, as mentioned here: https://github.com/fusionauth/fusionauth-issues/issues/1005

Reply:
@mooreds I think it would make sense to add the override section here, unless you think another place makes more sense? https://fusionauth.io/docs/v1/tech/email-templates/email-templates

Reply:
It is a premium feature, correct (only available with a license)? If is I'd add a separate page, as we'll want it to show up in the premium features section. Fine to link to it from https://fusionauth.io/docs/v1/tech/email-templates/email-templates

If not, https://fusionauth.io/docs/v1/tech/email-templates/email-templates is fine. Still think a separate page would make sense.